ChattoAdditions/Source/Input/ChatInputBar.swift

```diff
@@ -32,6 +32,8 @@ public protocol ChatInputBarDelegate: class {
     func inputBarSendButtonPressed(_ inputBar: ChatInputBar)
     func inputBar(_ inputBar: ChatInputBar, shouldFocusOnItem item: ChatInputItemProtocol) -> Bool
     func inputBar(_ inputBar: ChatInputBar, didReceiveFocusOnItem item: ChatInputItemProtocol)
+    func inputBar(_ inputBar: ChatInputBar, didShowPlaceholderWithText placeholderText: String?)
```

Comment:
You can take `placeholderText` from input bar itself. No need to pass it in delegate callback